src/domain/products/edit/sections/general/index.tsx

```diff
@@ -95,26 +95,46 @@ const GeneralSection = ({ product }: Props) => {
 
 type DetailProps = {
   title: string
-  value?: string | null
+  value?: string[] | string | null
 }
 
 const Detail = ({ title, value }: DetailProps) => {
+  const DetailValue = () => {
+    if (!Array.isArray(value)) {
+      return <p>{value ? value : "–"}</p>
+    }
+
+    if (value.length) {
+      return <DelimitedList list={value} delimit={2} />
+    }
+
+    return <p>–</p>
+  }
+
   return (
     <div className="flex items-center justify-between inter-base-regular text-grey-50">
```

Comment:
Suggestion: ESLint should now sort classes (you might need to restart the linting server)